sorc/link_workflow.sh

```diff
@@ -144,10 +144,15 @@ cd "${HOMEgfs}/parm/ufs" || exit 1
 ${LINK_OR_COPY} "${HOMEgfs}/sorc/ufs_model.fd/tests/parm/noahmptable.tbl" .
 
 cd "${HOMEgfs}/parm/post" || exit 1
-for file in params_grib2_tbl_new nam_micro_lookup.dat
+for file in params_grib2_tbl_new
 do
   ${LINK_OR_COPY} "${HOMEgfs}/sorc/upp.fd/parm/${file}" .
 done
+cd "${HOMEgfs}/parm/post" || exit 1
+for file in nam_micro_lookup.dat
+do
+  ${LINK_OR_COPY} "${HOMEgfs}/sorc/upp.fd/fix/${file}" .
```

Comment:
Since this is a UPP fix file, should this be linked in the `HOMEgfs/fix`? @DavidHuber-NOAA

Reply:
I don't think so.  All of the files in `HOMEgfs/fix` are linked to `glopara/fix` with the exception of those in `fix/product`, which are part of the global workflow and not linked in, so I'm not sure if that follows convention.  The only other case where we link in fix files from a submodule is the gsi-monitor to `HOMEgfs/parm/monitor`, so I think keeping it in parm is fine.

Reply:
works for me.